### PR TITLE
add coinbaseethdai feed

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: world/usd.go [hunk 1]
```diff
@@ -53,16 +53,17 @@ func (tw *TheWorld) getBitFinexInfo(url string) common.BitFinexData {
...
-		BinancePAX:   tw.getBinanceInfo(tw.endpoint.BinanceETHPAX()),
-		BinanceTUSD:  tw.getBinanceInfo(tw.endpoint.BinanceETHTUSD()),
-		BinanceUSDT:  tw.getBinanceInfo(tw.endpoint.BinanceETHUSDT()),
+		Timestamp:        0,
+		CoinbaseUSD:      tw.getCoinbaseInfo(tw.endpoint.CoinbaseETHUSD()),
+		GeminiUSD:        tw.getGeminiGoldInfo(),
+		CoinbaseUSDC:     tw.getCoinbaseInfo(tw.endpoint.CoinbaseETHUSDC()),
+		BinanceUSDC:      tw.getBinanceInfo(tw.endpoint.BinanceETHUSDC()),
+		CoinbaseDAI:      tw.getCoinbaseInfo(tw.endpoint.CoinbaseETHDAI()),
+		CoinbaseDAI10000: tw.getCoinbaseInfo(tw.endpoint.CoinbaseETHDAI10000()),
```
Comment (secmask) [inline world/usd.go]: did you verify that `CoinbaseDAI10000` has the same response format with `CoinbaseDAI`? we can use `getCoinbaseInfo` only if they are the same

File: feed-provider/cmd/main.go [hunk 2]
```diff
@@ -39,12 +39,12 @@ func run(c *cli.Context) error {
...
 	}()
 	zap.ReplaceGlobals(l.Desugar())
-	coinbaseETHDAIfetcher, err := coinbase.NewFetcherFromCli(c, l)
+	coinbaseETHDAI10000fetcher, err := coinbase.NewFetcherFromCli(c, l)
 	if err != nil {
 		return err
 	}
 	fetchers := map[string]fetcher.Fetcher{
-		"CoinbaseETHDAI": coinbaseETHDAIfetcher,
+		"CoinbaseETHDAI10000": coinbaseETHDAI10000fetcher,
```
Comment (secmask) [inline feed-provider/cmd/main.go]: avoid to change if it's not require here???

File: common/types.go [hunk 3]
```diff
@@ -702,11 +702,12 @@ type WorldEndpoints struct {
...
-	CoinbaseUSDC SiteConfig `json:"coinbase_usdc"`
-	BinanceUSDC  SiteConfig `json:"binance_usdc"`
-	CoinbaseUSD  SiteConfig `json:"coinbase_usd"`
-	CoinbaseDAI  SiteConfig `json:"coinbase_dai"`
-	HitDai       SiteConfig `json:"hit_dai"`
+	CoinbaseUSDC     SiteConfig `json:"coinbase_usdc"`
+	BinanceUSDC      SiteConfig `json:"binance_usdc"`
+	CoinbaseUSD      SiteConfig `json:"coinbase_usd"`
+	CoinbaseDAI      SiteConfig `json:"coinbase_dai"`
+	CoinbaseDAI10000 SiteConfig `json:"coinbase_dai_10000"`
```
Comment (secmask) [inline common/types.go]: please provide example config for `CoinbaseDAI10000` endpoint

File: feed-provider/cmd/main.go (hunk 2 above)
Comment (NNagato) [reply feed-provider/cmd/main.go]: Luke Tran tells me help him to change this

File: common/types.go (hunk 3 above)
Comment (NNagato) [reply common/types.go]: updated in https://github.com/KyberNetwork/reserve-data/pull/892/commits/9f2ca78a465249ba8e604c8c83f43191ef5f243e

File: world/usd.go (hunk 1 above)
Comment (NNagato) [reply world/usd.go]: updated in https://github.com/KyberNetwork/reserve-data/pull/892/commits/9f2ca78a465249ba8e604c8c83f43191ef5f243e

**Pull request merged**